logger/logger.go

```diff
@@ -248,10 +288,16 @@ func (l *Logger) CriticalD(title string, data map[string]interface{}) {
 // CounterD implements the method for the KayveeLogger interface.
 // Logs with type = gauge, and value = value
 func (l *Logger) CounterD(title string, value int, data map[string]interface{}) {
-	data["title"] = title
-	data["value"] = value
-	data["type"] = "counter"
-	l.logWithLevel(Info, data)
+	if l.metricsOutput == OTLMetrics {
+		meter := global.Meter(fmt.Sprintf("%s", l.globals["source"]))
```

Comment:
It's actually safe to read but if someone changed the source while a concurrent counter call was happening it could potentially end up in a different source metric.  I will add the read lock